src/LtiAdvantage/AssignmentGradeServices/Result.cs

```diff
@@ -28,13 +28,13 @@ public class Result
         /// Maximum result score.
         /// </summary>
         [JsonProperty("resultMaximum")]
-        public double ResultMaximum { get; set; }
+        public double? ResultMaximum { get; set; }
```

Comment:
Is there nowhere that needs to adjust to deal with this? If not what is the expected behavior of a null of both of these fields.

Reply:
Nothing more to change here, but we'll need to adjust our usages in Panopto when we upgrade package version.
Re behavior, from the [spec](https://www.imsglobal.org/spec/lti-ags/v2p0/#resultscore):

> 2.3.4.5 resultScore
> The current score for this user. The value must be a numeric value. If no value exists, this attribute may be omitted, or have an explicit null value.
> 
> 2.3.4.6 resultMaximum
> The 'resultMaximum' value MUST be a positive number (with 0 considered a negative number); if no value is specified, then a default maximum value of 1 must be used.

We currently have a [workaround](https://github.com/Panopto/panopto-core/blob/32dab72f9f22b3b5c8aaf86a48f95b31e23fc4e2/Panopto/Integrations/IntegrationsLib/GradebookReporting/LTI13AGSReporter.cs#L278) in place for `resultMaximum` that this change will allow us to fix.  This will also allow us to resolve the deserialization issue ([TFS-123946](http://tfs.panopto.local:8080/tfs/DefaultCollection/Panopto%20Current/_workitems/edit/123946)) due to results containing null/missing `resultScore`.

Reply:
Sounds good